src/store/health/actions.js

```diff
@@ -43,29 +53,28 @@ function checkGraphHealth() {
     try {
       const data = await graphRequest(endpoint, query);
 
+      console.log(data) // eslint-disable-line
+
       const {
-        entityCount,
-        failed,
+        fatalError,
         synced,
-        latestEthereumBlockNumber,
-        totalEthereumBlocksCount,
-      } = data.subgraphs[0].currentVersion.deployment;
+        chains,
+      } = data.indexingStatusForCurrentVersion;
 
       dispatch({
         type: ActionTypes.HEALTH_UPDATE_SERVICE,
         meta: {
           serviceName: 'graph',
           state: {
             isReachable: true,
-            entityCount,
```

Comment:
`entityCount` is being displayed in the dashboard, we would need to either find out if we can still get this number somewhere or delete the UI for it